### PR TITLE
add slice and more information about charts via link in readme

### PR DESCRIPTION
**Pull request by Guseyn**
regarding to #1

### CODE REVIEW

**Comment by Guseyn** [comment]
@g4s8 could you make progress on this pr?

File: src/main/java/com/artipie/helm/HelmSlice.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import java.util.HashSet;
+import java.util.Map;
+import org.reactivestreams.Publisher;
+
+/**
+ * HelmSlice.
+ *
+ * @since 0.1
+ */
+public final class HelmSlice implements Slice {
```
Comment (g4s8) [inline src/main/java/com/artipie/helm/HelmSlice.java:41]: @Guseyn I think we need an integration test as well as a `Slice`

Comment (Guseyn) [reply src/main/java/com/artipie/helm/HelmSlice.java:41]: @g4s8 do you know any example of such skeleton for integration test?

Comment (g4s8) [reply src/main/java/com/artipie/helm/HelmSlice.java:41]: @Guseyn it should have a name ended with `ITCase` to be properly processed by Maven failsafe plugin, e.g. `HelmSliceITCase`

**Comment by Guseyn** [comment]
@g4s8 plz check the updates

File: src/main/java/com/artipie/helm/HelmSlice.java (hunk 1 above)
Comment (Guseyn) [reply src/main/java/com/artipie/helm/HelmSlice.java:41]: @g4s8 done

**Pull request merged**